src/pages/Login.jsx

```
@@ -0,0 +1,303 @@
+    },
+    [form.pw.value]
+  ); // pwConfirm 검사를 위해 form.pw.value를 의존성으로 추가
+
+  const handleChange = useCallback(
+    (e) => {
+      const { name, value } = e.target;
+      const { isValid, message } = validateField(name, value);
+      setForm((prevForm) => ({
+        ...prevForm,
+        [name]: { value, message, isValid },
+        // 비밀번호가 변경되면 비밀번호 확인 필드도 다시 검사
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)